src/main/java/seedu/duke/exercises/Exercise.java

```diff
@@ -3,21 +3,39 @@
 import seedu.duke.storage.models.ExerciseModel;
 import seedu.duke.storage.models.WorkoutModel;
 
+import java.util.logging.Logger;
+
+import static seedu.duke.logger.LoggerUtil.setupLogger;
+
+/**
+ * Represents an exercise inside a workout.
+ * It contains basic information of an exercise such as reps, sets and status.
+ */
 public class Exercise {
     protected String description;
-
     //protected String muscle;
     protected int sets;
     protected int reps;
     protected Boolean isDone = false;
+    static final Logger LOGGER = Logger.getLogger(Exercise.class.getName());
 
     public Exercise(String description, int sets, int reps) {
         this.description = description;
         this.sets = sets;
         this.reps = reps;
     }
 
+    public Exercise() {
+        setupLogger(LOGGER);
```

Comment:
I think this can be put in the constructor above. Only the constructor above will be called in the rest of our code which means the logger will never be initialised.

Reply:
Yes just rechecked my log file, you are right. Could you amend the change when fixing your issue?